backend/mora/db_extensions.json

```diff
@@ -2,7 +2,8 @@
   "organisationfunktion": {
     "attributter": {
       "udvidelser": [
-        "primær"
+        "primær",
+        "fraktion"
```

Comment:
Wouldn't it make sense to use a number rather than a string for this?

Reply:
The field is meant to contain things other than numbers as well.